### PR TITLE
Reworking Drug Response Importers

### PR DESCRIPTION
**Pull request by kellrott**

### CODE REVIEW

File: tests/unit/ccle/test_drug_response.py [hunk 1]
```diff
@@ -35,23 +35,23 @@ def validate(helpers, emitter_directory, biosample_path, drug_response_path):
...
+    helpers.assert_edge_file_valid(DrugResponse, Compound, response_to_file)
     helpers.assert_vertex_file_valid(Compound, compound_file)
     helpers.assert_edge_joins_valid(all_files, exclude_labels=['Aliquot'])
     # missing vertexes
-    for f in ['drug_response.Aliquot.Vertex.json.gz', 'drug_response.Biosample.Vertex.json.gz',
-              'drug_response.Individual.Vertex.json.gz', 'drug_response.Project.Vertex.json.gz']:
-        v = eval(f.split('.')[1])
-        f = os.path.join(emitter_directory, f)
-        helpers.assert_vertex_file_valid(v, f)
+    # for f in ['drug_response.Aliquot.Vertex.json.gz', 'drug_response.Biosample.Vertex.json.gz',
```
Comment (adamstruck) [inline tests/unit/ccle/test_drug_response.py]: Why are these tests commented out?

File: transform/ccle/drug_response.py [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
-import logging
 import glob
 import ujson
 from types import SimpleNamespace as SN
 
 import bmeg.ioutils
 from bmeg.emitter import JSONEmitter
-from bmeg.vertex import ParamacalogicalProfile, Aliquot, Biosample, Individual, Project
-from bmeg.edge import ParamacalogicalProfileIn, ResponseTo, AliquotFor, BiosampleFor, InProject
+from bmeg.vertex import DrugResponse, Aliquot  # , Biosample, Individual, Project
```
Comment (adamstruck) [inline transform/ccle/drug_response.py]: Please remove the commented out code if it is not needed.

File: transform/ctrp/ctrp-convert.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+from bmeg.enrichers.drug_enricher import compound_factory
+
+from bmeg.emitter import JSONEmitter
+
+data_dir = sys.argv[1]
+
+base_dir = os.path.dirname(__file__)
+
+ccl_table_path = os.path.join(base_dir, "ctrp-cellline.table")
```
Comment (adamstruck) [inline transform/ctrp/ctrp-convert.py]: Where did this file come from? Should there be a DVC source entry for it?

File: transform/ccle/drug_response.py [hunk 4]
```diff
@@ -112,6 +94,7 @@ def transform(biosample_path='outputs/ccle/Biosample.Vertex.json*',
             compound.gid(),
         )
 
+    """
```
Comment (adamstruck) [inline transform/ccle/drug_response.py]: Is this commented block of code needed?

**Comment by adamstruck** [comment]
I'm finding some connectivity issues... Lots of `DrugResponseIn` edges point to `DrugResponse` vertices that don't exist. Here is the gid of one of the missing vertices: `DrugResponse:(5Z)-7-Oxozeaenol:ACH-000956`

File: transform/ctrp/ctrp-convert.py (hunk 3 above)
Comment (bwalsh) [reply transform/ctrp/ctrp-convert.py]: talked to kyle about this.  there is a shell script that creates it.  afaik, it is checked into git.
long term, we should deprecate the name munging that is happening in various transforms and maintain this file manually in git.

**Pull request merged**